services/director-v2/src/simcore_service_director_v2/modules/instrumentation/_models.py

```diff
@@ -88,7 +88,7 @@ def __post_init__(self) -> None:
             "time to pull docker images",
             labelnames=_INSTRUMENTATION_LABELS,
             namespace=_METRICS_NAMESPACE,
-            buckets=_RATE_BPS_BUCKETS,
+            buckets=_BUCKETS_TIME_S,  # SAN: the previous version was wrong, am I correct?
```

Comment:
@sanderegg could you please check the previous version, as I think it was wrong?

Reply:
since this is a duration, you are correct that it was wrong before. good catch, and good to know.